plugin/src/main/java/com/iridium/iridiumcore/utils/SkinUtils.java

```diff
@@ -96,4 +101,20 @@ private static String getURLContent(String urlStr) throws UnknownHostException {
         }
         return stringBuilder.toString();
     }
+
+    private static boolean validateUsername(String input) {
+        if (input == null || input.isEmpty() || input.length() > 16) {
+            return false;
+        }
+
+        for (int i =0, len = input.length(); i < len; i++) {
+            char c = input.charAt(i);
+            if ((c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z') || (c >= '0' && c <= '9') || (c == '_')) {
+                continue;
+            }
+            return false;
+        }
+
+        return true;
+    }
```

Comment:
This can be shortened + improved. You should also rename the method to `isValidUsername` or something similar.
```suggestion
    private static boolean validateUsername(String input) {
        return input != null && input.matches("\\w{3,16}");
    }
```